README.md

```diff
@@ -27,3 +27,22 @@ Can be used to ensure code changes are compilable, `-Djboss.home=foo` is workaro
 ```
 mvn clean verify -DskipTests -DskipITs -Djboss.home=foo
 ```
+
+## Zip distribution bundle
+Distribution bundle contains this testsuite, pre-loaded local maven repository and dump of Docker images used in tests.
+Creation of the `eap-microprofile-test-suite-dist.zip` bundle is managed via `./mp.sh` script.
+This script assumes that all the necessary artifacts present in `${user.home}/.m2/repository/`, in other words that the TS was compiled and executed on local machine prior the bundle creation.
+
+Bundle creation:
+```
+./mp.sh dist-zip
+```
+
+Execution on target machine:
```

Comment:
There should be added note about the need to have JDK 8+ and docker daemon/docker=>podman installed on the machine as it looks like that nothing more is needed.

Reply:
why? 

these are TS prerequisites, not exclusive to zip distro

Created https://github.com/jboss-eap-qe/eap-microprofile-test-suite/issues/52 for it

Reply:
When running TS using bundle then Maven is not required to be installed on the machine, right? If yes pre-requites for running from bundle will be different and it should be pointed out.